app/controllers/admin/edition_change_notes_controller.rb

```diff
@@ -0,0 +1,76 @@
+class Admin::EditionChangeNotesController < Admin::BaseController
+  before_action :find_edition
+  before_action :enforce_permissions!
+  before_action :limit_edition_access!
+  layout "design_system"
+
+  def index
+    @change_notes = @edition
+      .document
+      .editions
+      .without_not_published
+      .where(minor_change: false)
+      .select(:id, :major_change_published_at, :change_note)
+  end
+
+  def edit
+    # This is the edition the change note was originally set on, not the latest edition
+    @edition_to_change = Edition.find(params[:id])
+  end
+
+  def update
+    edition_to_change = Edition.find(params[:id])
+    old_change_note = edition_to_change.change_note
+
+    # rubocop:disable Rails/SkipsModelValidations
+    edition_to_change.update_attribute(:change_note, params[:new_change_note])
```

Comment:
> Just a question. We have no validations here. Should we check for presence of input, or are we happy to trust the user since it's a GDS Admin?

I left the validations out as this is just for internal use.  We'd need to review the validations if we were to expand the usage.

> One other question. On second line i've had to change a minor change to a major change before and add a change note. Is that out of scope for this card?

I started writing code to do this, then realised it got more complicated as you had to also request a date/time from the user.  I didn't think I'd get that completed this week, so put it out of scope.  I definitely think this is something we should add in any future iterations.